roles/docker-garbage-collection/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "create docker_garbage_collection directory"
+  file:
+    path: "{{garbage_home}}"
+    state: "directory"
+    owner: "{{garbage_user}}"
+    group: "{{garbage_group}}"
+    mode: "0754"
+
+- name: "create image backup directory"
+  file:
+    path: "{{backup_images}}"
+    state: "directory"
+    owner: "{{garbage_user}}"
+    group: "{{garbage_group}}"
+    mode: "0754"
```

Comment:
These tasks can be combined into one. For example: https://github.com/osu-mist/ansible-roles/blob/master/roles/api-environment/tasks/express-api.yml#L2-L12